packages/scrum-client/src/App.css

```diff
@@ -1,22 +1,12 @@
 @import url('https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600,700,800');
 
-* {
-  box-sizing: border-box;
-}
-
-h1,h2,h3,h4,h5,h6,label,span {
```

Comment:
I think we should create our own bootstrap theme

Reply:
Thats a great idea 🥇